react/react1/week1/todo-list/src/components/toDoList.jsx

```diff
@@ -0,0 +1,21 @@
+import todos from '../data/toDos';
+import ToDoItem from './toDoItem';
+
+export default function ToDoList() {
+    return (
+        <div className='toDoList'>
+            <ul>
+                {todos.map((todo) => {
+                    return (
+                        <ToDoItem
+                            key={todos.indexOf(todo) + 1}
+                            description={todo.description}
```

Comment:
`todo={todo}` would do the job and you could access its props inside `TodoItem.jsx` :)

Reply:
Thank you  Anna for your feed back! I appreciated it :) I will make changes :)